### ISSUE TITLE
When will Sponsored Reserves be fully implemented?

### ISSUE THREAD

**Issue by shredding** [Soneso/stellar-ios-mac-sdk#106]
The sponsored reserves implementation is not working.

Given this xdr:

```
AAAAAgAAAAA0jCzxfXQz6m2qbnLP+65jGoAlFe4iq6PDscyjMQFpaQAtxsAB6r3KAAAB3AAAAAEAAAAAAAAAAAAAAABgJQcCAAAAAAAAAAMAAAABAAAAAB4iR3Mb6J8v89M2SdB9HqBrRdxzyIS66zoSEI3YmXjBAAAAEAAAAACHq6ZMhPkTrMxCxpHMRzP0+x9RiQrWytu63h+8FZ9paAAAAAEAAAAAh6umTIT5E6zMQsaRzEcz9PsfUYkK1srbut4fvBWfaWgAAAAGAAAAAVVTREMAAAAAO5kROA7+mIugqJAOsc/kTzZvfb6Ua+0HckD39iTfFcV//////////wAAAAEAAAAAh6umTIT5E6zMQsaRzEcz9PsfUYkK1srbut4fvBWfaWgAAAARAAAAAAAAAAHYmXjBAAAAQFiyng7KylEQ2YEAL+mmEmPx4V0WipbQy6kuc/RDVjGVsjW45L1FQbGU7eja7gAIsaU0m83xW1eGQn/m1A5RTww=
```

... and do ...

```
try Transaction.init(envelopeXdr: xdr)
```

We get ...

```
Unknown operation body beginSponsoringFutureReserves(stellarsdk.BeginSponsoringFutureReservesOpXDR(sponsoredId: stellarsdk.PublicKey))
```

This is because the support for beginning and ending sponsoring future reserves is missing in the [Operation](https://github.com/Soneso/stellar-ios-mac-sdk/blob/a561b02677ad29fb6182c65e4da8042676af20f8/stellarsdk/stellarsdk/sdk/Operation.swift) class.

I tried to make it work, but it looks like the entire implementation is a WIP, e.g. [this](https://github.com/Soneso/stellar-ios-mac-sdk/blob/master/stellarsdk/stellarsdk/sdk/EndSponsoringFutureReservesOperation.swift) looks like a copy and paste from `BeginSponsoringFutureReservesOperation` and is not working either.

**Comment by christian-rogobete** [comment]
fixed in [release 1.9.5](https://github.com/Soneso/stellar-ios-mac-sdk/releases/tag/1.9.5)

**Comment by shredding** [comment]
Wow, that was super fast. Thank you so much.

Verified, it works.
